connectors/src/resources/connector_resource.ts

```diff
@@ -55,14 +69,17 @@ export class ConnectorResource extends BaseResource<ConnectorModel> {
         { transaction }
       );
 
-      const connectorRes = new this(ConnectorModel, connector.get());
+      const strategy = getConnectorProviderStrategy(type);
 
-      await connectorRes.providerStrategy.makeNew(
-        connectorRes,
+      const configuration = await strategy.makeNew(
```

Comment:
We could keep the previous code since we don't need the strategy before creating the connector resource?

Reply:
yep multi refactor side effect